io_batch_import_objs.py

```diff
@@ -31,6 +31,7 @@
 
 
 import bpy
+import os
```

Comment:
As of Python 3.4 there is a new pathlib module and my idea was actually to get rid of the os module used in my original version. I don't think it is necessary to import an entire library for only one function call. Or is there no substitute?